hack/overlays/overlay-envoy.yaml

```diff
@@ -0,0 +1,54 @@
+#@ load("@ytt:overlay", "overlay")
+#@ load("@ytt:data", "data")
+
+#! privatize loadbalancer, disable hostport and rewrite command
+
+#@ if data.values.namespace == "contour-internal":
+#@overlay/match by=overlay.subset({"kind":"Service","metadata":{"name": "envoy"}})
+---
+spec:
+  #@overlay/remove
+  externalTrafficPolicy: Local
+  #@overlay/match
+  type: ClusterIP
+#@ end
+
+#@overlay/match by=overlay.subset({"kind":"DaemonSet","metadata":{"name": "envoy"}})
+#@overlay/match-child-defaults when=1
```

Comment:
Since the match above doesn't have an `expects` set I'm assuming it defaults to `1`

Does that mean `when=1` here is redundant?

Reply:
Your are right, it's redundant :). Will remove it